### ISSUE TITLE
When searching for phrases/words on the public site, I want the relevant results highlighted on the individual document detail pages

### ISSUE THREAD

**Issue by kseniaryzhova** [Princeton-CDH/geniza#1368]
**Is your feature request related to a problem? Please describe.**
Right now, searching for phrases and words in transcriptions or descriptions highlights the relevant phrases/words in the thumbnail of each document in the search results page. When you click into an individual document detail page, the highlight disappears and you have to search the page all over again.

**Describe the solution you'd like**
We would like the relevant word or phrase to remain highlighted in the document detail page after we click into a record.

**Comment by kseniaryzhova** [comment]
@blms @cblissg MR has noted that in light mode, the search results highlighted view is very hard to see (she finds it hard to distinguish between black and green) so could we lighten the green? Or make the highlight clearer?

**Comment by blms** [comment]
@kseniaryzhova Yes but I would consider that a different issue, since this refers to showing highlighted search terms in the document page specifically. Can you open up a new issue for it?
